src/ui/server/controllers/insurance/policy-and-export/about-goods-or-services/save-and-back/index.ts

```diff
@@ -26,7 +28,9 @@ export const post = async (req: Request, res: Response) => {
     const { referenceNumber } = req.params;
 
     if (hasFormData(req.body)) {
-      const saveResponse = await callMapAndSave(req.body, application, generateValidationErrors(req.body));
+      const payload = constructPayload(req.body, FIELD_IDS);
+
+      const saveResponse = await callMapAndSave(payload, application, generateValidationErrors(payload));
```

Comment:
in other places, we are generateValidationErrors from req.body

Reply:
All the other PRs are using the payload in `generateValidationErrors` like this - I think as soon as a `req.body` is received, it should be passed to the payload function and only consumed this way. Otherwise the validation function(s) could be consuming bad/invalid/irrelevant data.